Tests/Functional/ParameterEscapingTest.php

```diff
@@ -32,176 +38,186 @@ public function setUp(): void
             '<test:contentSlot content="{safeVariable -> f:format.raw()}" />',
-            "<div>Pre-rendered output without unsafe user input</div>\n"
+            "<div>Pre-rendered output without unsafe user input</div>\n",
         ];
 
         // <fc:slot name="slot" /> in component [escape=true]
         yield [
             '<test:SlotParameter slot="<b>some html</b>" />',
-            "<b>some html</b>\n"
+            "<b>some html</b>\n",
         ];
         yield [
             '<test:SlotParameter><fc:content slot="slot"><b>some html</b></fc:content></test:SlotParameter>',
-            "<b>some html</b>\n"
+            "<b>some html</b>\n",
         ];
         yield [
             '<test:SlotParameter slot="{maliciousVariable}" />',
-            "&lt;script&gt;alert(&#039;This JavaScript should not be executed by the browser&#039;)&lt;/script&gt;\n"
+            "&lt;script&gt;alert(&#039;This JavaScript should not be executed by the browser&#039;)&lt;/script&gt;\n",
         ];
         yield [
             '<test:SlotParameter><fc:content slot="slot">{maliciousVariable}</fc:content></test:SlotParameter>',
-            "&lt;script&gt;alert(&#039;This JavaScript should not be executed by the browser&#039;)&lt;/script&gt;\n"
+            "&lt;script&gt;alert(&#039;This JavaScript should not be executed by the browser&#039;)&lt;/script&gt;\n",
         ];
         yield [
             '<test:SlotParameter slot="{safeVariable -> f:format.raw()}" />',
-            "<div>Pre-rendered output without unsafe user input</div>\n"
+            "<div>Pre-rendered output without unsafe user input</div>\n",
         ];
         yield [
             '<test:SlotParameter><fc:content slot="slot">{safeVariable -> f:format.raw()}</fc:content></test:SlotParameter>',
-            "<div>Pre-rendered output without unsafe user input</div>\n"
+            "<div>Pre-rendered output without unsafe user input</div>\n",
         ];
 
         // {param} in component [escape=false]
         yield [
             '<test:StringParameter param="<b>some html</b>" />',
-            "&lt;b&gt;some html&lt;/b&gt;\n"
+            "&lt;b&gt;some html&lt;/b&gt;\n",
         ];
         yield [
             '<test:StringParameter param="{maliciousVariable}" />',
-            "&lt;script&gt;alert(&#039;This JavaScript should not be executed by the browser&#039;)&lt;/script&gt;\n"
+            "&lt;script&gt;alert(&#039;This JavaScript should not be executed by the browser&#039;)&lt;/script&gt;\n",
         ];
         yield [
             '<test:StringParameter param="{safeVariable -> f:format.raw()}" />',
-            "&lt;div&gt;Pre-rendered output without unsafe user input&lt;/div&gt;\n"
+            "&lt;div&gt;Pre-rendered output without unsafe user input&lt;/div&gt;\n",
         ];
     }
 
     #[Test]
     #[DataProvider('renderDataProvider')]
     public function render(string $template, string $expected): void
     {
-        $view = new TemplateView();
```

Comment:
You could probably also use `TemplateView` from Fluid Standalone here, which would avoid the version switch. Same goes for all other test methods

Reply:
Oh that's possible :exploding_head: